server/views/formPages/recat/securityBack.html

```diff
@@ -14,14 +14,17 @@
 
 <h1 id="securityInputHeader" class="govuk-heading-l">{{ pageTitle }}</h1>
 
-  {% set warningText %}
-    This prisoner was referred to the security team
-  {% endset %}
-  {{ govukWarningText({
-    text: warningText,
-    iconFallbackText: "Warning",
-    classes: "forms-warning-text warning-overrides"
-  }) }}
+  {% if data.recat.securityInput.securityInputNeededText %}
```

Comment:
i think the condition is still wrong.
SecurityInputNeededText will be populated only when you say Yes to securityNoteNeeded.
So it might be like this
if(SecurityNoteNeeded == No){
    Note Was added
}elseif securityInputNeededText{
    show securityInputNeededText.
}

in terms of auto and flagged scenarios for security neither securityNoteNeeded nor securityInputNeededText will be populated.

Reply:
Just tested - the string in the database 
{"recat": {"oasysInput": {"date": "24/09/2019", "oasysRelevantInfo": "No"}, **"securityInput": {"securityInputNeeded": "Yes", "securityInputNeededText": "This kind of competence only comes from practice."}**, "prisonerBackground": {"offenceDetails": "no"}}, **"security": {"review": {"securityReview": "Furious and Riddick movies these days, so a brand-new franchise would help him out."}}**}
Allows the change to be backwards compatible with existing cases that don't have the **securityNoteNeeded** variable

Reply:
Also modified two cases with:
-"securityInput": {"securityNoteNeeded": "", "securityInputNeeded": "Yes"}
- "securityInput": {"securityNoteNeeded": "", "securityInputNeeded": ""} 
And it works

Reply:
how about when there is no security input at all for RECAT as during flagged and auto there will be no 
"securityInput": {"securityNoteNeeded": "", "securityInputNeeded": ""}
can we also test if status is AUTO and there is no securityInput element in json at all

Reply:
These scenarios have already been tested.

Reply:
**transferToSecurity - true**
{"recat": {"securityBack": {}}, "security": {"review": {"securityReview": "Despite the violence and chaos and angry debate in Parliament, the measure passed easily, mostly because the ministers who were opposed either resigned or were forced out of the country's two ruling coalition parties."}}}

{"socProfile": {"nomsId": "G2843GF", "riskType": "SOC", "transferToSecurity": true, "provisionalCategorisation": "C"}, "extremismProfile": {"nomsId": "G2843GF", "riskType": "EXTREMISM", "notifyRegionalCTLead": false, "increasedRiskOfExtremism": false, "provisionalCategorisation": "C"}}

Reply:
LGTM